apps/webapp/src/fetchers/stream.ts

```diff
@@ -65,7 +65,9 @@ export const streamToPromise = <T>(query: AsyncIterable<T>): Promise<T[]> => {
         }
         resolve(result);
       } catch (e) {
-        reject(e);
+        if (e instanceof Error) reject(e);
+        else if (typeof e === 'string') reject(new Error(e));
+        else reject(new Error('Unknown error in `streamToPromise`'));
```

Comment:
Due to [the aforementioned ESLint rule](https://eslint.org/docs/latest/rules/prefer-promise-reject-errors) about only rejecting with an instance of `Error`, I had to rework this `catch` clause a bit to ensure that it's actually rejecting an `Error` instance. I figured that the `Error`/`string` cases should catch most errors; anything else, the developer encountering this error will have to dig a bit deeper.

I don't know when this would even get triggered, since this is related to some view server stuff that I'm less familiar with. But if devs have suggestions, I'm all 👂🏻 👂🏻